src/test/java/sorinfoca/Tests/Tema6/HerokuForms.java

```diff
@@ -0,0 +1,77 @@
+package sorinfoca.Tests.Tema6;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import sorinfoca.driver.BrowserManager;
+import java.io.File;
+import java.io.IOException;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+    private static File screenshot;
+
+    public static void main(String[] args) throws InterruptedException {
+        navigateToHerokuHomePage();
+        try {
+            fillOutForm();
+            scrollToSubmitButton();
+            submitForm();
+        } catch (NoAlertPresentException e) {
+            sorinfoca.utils.FileUtils.takeScreenshot(driver, "alert");
+            System.out.println("No alert present");
+            try {
```

Comment:
You don't need anymore this try - catch block inside the first catch. Please remove it. Thanks!